contracts/chain-adapters/Arbitrum_CustomGasToken_L2_Forwarder.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { CircleCCTPAdapter, CircleDomainIds } from "../libraries/CircleCCTPAdapter.sol";
+import { ITokenMessenger as ICCTPTokenMessenger } from "../external/interfaces/CCTPInterfaces.sol";
+import { ArbitrumCustomGasTokenInbox, ArbitrumL1ERC20GatewayLike } from "../interfaces/ArbitrumBridgeInterfaces.sol";
+import { ForwarderBase } from "./ForwarderBase.sol";
+import { Arbitrum_CustomGasToken_AdapterBase, FunderInterface } from "./Arbitrum_CustomGasToken_AdapterBase.sol";
+
+/**
+ * @notice Contract containing logic to send messages from Arbitrum to an AVM L3.
```

Comment:
In all contracts, its helpful to note which layer this contract is supposed to be deployed on. In this case, this contract is supposed to be deployed on L2, right?

Reply:
Yes, it is L2: https://github.com/across-protocol/contracts/pull/610/commits/05dfc6eefae26f129ab68223a011766894ec144b